scripts/wrist_test.py

```diff
@@ -0,0 +1,245 @@
+
+            for t in targets:
+                msg.position = [t, t]
+                publisher.publish(msg)
+                rospy.sleep(0.01)
+
+
+def move_all(move_type="sequentially", shoulder_range=None, elbow_range=None, wrist_range=None, head_range=None):
+    shoulder_range = [0.0, -1.1] if not shoulder_range else shoulder_range
+    elbow_range = [0.0, 0.6] if not elbow_range else elbow_range
+    wrist_range = [-1.5, -0.05] if not wrist_range else wrist_range
+    head_range = [-0.56, 0.56] if not head_range else head_range
+
+    msg = JointState()
+    msg.name = ["shoulder_left_axis0", "elbow_left_axis0", "wrist_left_axis0", "wrist_left_axis1",
+                "shoulder_right_axis0", "elbow_right_axis0", "wrist_right_axis0", "wrist_right_axis1",
+                "head_axis0"]
+
+    if move_type == "sequentially":
+        targets = np.linspace(shoulder_range[0], shoulder_range[1], 200)
+        targets_el = np.linspace(elbow_range[0], elbow_range[1], 200)
+        targets_wr = np.linspace(wrist_range[0], wrist_range[1], 200)
+        targets_head = np.linspace(head_range[0], head_range[1], 200)
+    elif move_type == "randomly":
+        targets = np.random.uniform(shoulder_range[0], shoulder_range[1], 200)
+        targets_el = np.random.uniform(elbow_range[0], elbow_range[1], 200)
+        targets_wr = np.random.uniform(wrist_range[0], wrist_range[1], 200)
+        targets_head = np.random.uniform(head_range[0], head_range[1], 200)
+    elif move_type == "randomly_in_range":
+        shoulder_range = [np.random.uniform(shoulder_range[0], shoulder_range[1] / 4),
+                          np.random.uniform(shoulder_range[0] / 2, shoulder_range[1])]
+        elbow_range = [np.random.uniform(elbow_range[0], elbow_range[1] / 4),
+                       np.random.uniform(elbow_range[0] / 2, elbow_range[1])]
+        wrist_range = [np.random.uniform(wrist_range[0], wrist_range[1] / 4),
+                       np.random.uniform(wrist_range[0] / 2, wrist_range[1])]
+        head_range = [np.random.uniform(head_range[0], head_range[1] / 4),
+                      np.random.uniform(head_range[0] / 2, head_range[1])]
+        targets = np.linspace(shoulder_range[0], shoulder_range[1], 200)
+        targets_el = np.linspace(elbow_range[0], elbow_range[1], 200)
+        targets_wr = np.linspace(wrist_range[0], wrist_range[1], 200)
+        targets_head = np.linspace(head_range[0], head_range[1], 200)
+
+    for t, te, tw, th in zip(targets, targets_el, targets_wr, targets_head):
+        msg.position = [t, te, tw, tw, t, te, tw, tw, th]
+        msg.velocity = [0] * len(msg.name)
+        msg.effort = [0] * len(msg.name)
+        publisher.publish(msg)
+        rate.sleep()
+
+
+def test_scene_1():
+    rospy.loginfo("########## SCENE 1 ##########")
+    rospy.loginfo("Description: Only move the wrist")
+    hard_reset()
+    rospy.loginfo("Move left wrist")
+    move_wrist("left", 2)
```

Comment:
same here. dont use strings. they are subject to error. use enum or const strings definitions

Reply:
Done